contracts/FastBreakV1.cdc

```diff
@@ -13,47 +13,52 @@
 
 import NonFungibleToken from 0xNFTADDRESS
 import TopShot from 0xTOPSHOTADDRESS
+import MetadataViews from 0xMETADATAVIEWSADDRESS
 
 /// Game & Oracle Contract for Fast Break V1
 ///
-pub contract FastBreakV1: NonFungibleToken {
+access(all) contract FastBreakV1: NonFungibleToken {
+
+    access(all) entitlement Play
+    access(all) entitlement Create
+    access(all) entitlement Update
 
     /// Contract events
     ///
-    pub event ContractInitialized()
+    access(all) event ContractInitialized()
```

Comment:
This can be removed

Reply:
Do you mean completely removing them due to https://cadence-lang.org/docs/language/core-events#account-contract-added ?